base/lnls-prune-artifacts.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+set -eu
+
+
+filter_out_dirs() {
+    list="$1"
+    exclude_list="$2"
+
+    for item in $exclude_list; do
+        dir=$item
+
+        while [ "$dir" != "/" ]; do
+            list=$(echo "$list" | grep -xv $dir)
+
+            dir=$(dirname $dir)
+        done
```

Comment:
Yeah, I turned out to do so when testing to avoid infinite loops, but didn't get to a final approach on how to handle it.

Should we abort in such cases, or simply ignore that entry?

Reply:
IMO abort, it's breaking an expected invariant.